project/Build.scala

```diff
@@ -1833,6 +1833,94 @@ object Build {
       target := target.value / "scala3-library-bootstrapped",
     )
 
+  /* Configuration of the org.scala-js:scalajs-scalalib_2.13:*.**.**-bootstrapped project */
+  lazy val `scala-library-sjs` = project.in(file("library-js"))
+    // We add a dependency to the JVM library to have the classfile available
+    // (as they are not part of this artifact)
+    .dependsOn(`scala3-library-bootstrapped-new`)
+    .settings(
+      name          := "scala-library-sjs",
+      organization  := "org.scala-js",
+      moduleName    := "scalajs-scalalib_2.13",
+      version       := dottyVersion,
+      versionScheme := Some("semver-spec"),
+      // This is very tricky here since this is a Scala 3 project, but to be able to smoothly
+      // migrate the ecosystem, we need to be able to evict the Scala 2 library from the classpath.
+      // The problem is that the Scala 2 library for Scala.js has a _2.13 in the module's name, so we need
+      // to release Scala 3 for Scala.js with the same _2.13 instead of the _3.
+      // Yes, I know, this is weird and feels wrong.
```

Comment:
Put the comment *above* those three lines? It seems they target those three, but they are below, unlike all other comments, which is confusing.